walkthroughs/howto-cross-account/vars.env

```diff
@@ -4,10 +4,10 @@ export AWS_PRIMARY_ACCOUNT_ID=
 export AWS_SECONDARY_ACCOUNT_ID=
 export AWS_PRIMARY_PROFILE=primary
 export AWS_SECONDARY_PROFILE=secondary
-export PROJECT_NAME=cross-account
-export AWS_DEFAULT_REGION=us-west-2
+export PROJECT_NAME=
+export AWS_DEFAULT_REGION=
 export KEY_PAIR=
 
-export ENVOY_IMAGE=840364872350.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/aws-appmesh-envoy:v1.12.2.1-prod
+export ENVOY_IMAGE=840364872350.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/aws-appmesh-envoy:v1.15.1.0-prod
 export BACKEND_1_IMAGE=dockercloud/hello-world
 export BACKEND_2_IMAGE=karthequian/helloworld
```

Comment:
Why do we have these vars to begin with? None of our other projects use this method (AFAIK). Do we not instruct the user to populate these in the README?

Reply:
Our other walkthroughs do not have this `vars.env` file. But in this walkthrough, the README has instructions to update the `vars.env` and add the appropriate account and profile settings.

Reply:
I don't love this, but I won't block on it since we are not concerned about updating this walk through with this PR